src/main/resources/application.yaml

```diff
@@ -209,6 +209,13 @@ govnotify:
     applicationRefusedApplicant: ab6b3c49-7f7e-407a-b46d-8dda10e33803
     applicationRefusedOtherParty: 59ec29bf-5d40-4a97-bd54-cf7262b65185
 
+    applicationReheardApplicant: ac94290a-bc6a-4e3b-a9cb-34d48400450c
```

Comment:
I think we need to apply same formatting as the other notification templates here. For example:

> applicationReheard:
>     applicant: <template>
>     otherParty: <template>

Reply:
This has to go through at the moment. I am refactoring it on RIA-2975